ToDoList/ToDoList/Controller/LogListViewController.swift

```diff
@@ -0,0 +1,55 @@
+import UIKit
+
+class LogListViewController: UIViewController {
+    
+    private let logManager: LogManager
+    private var mainViewController: MainViewController? = nil
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    init(coder: MainViewController) {
+        self.logManager = LogManager()
+        self.mainViewController = coder
+        super.init(nibName: "LogListView", bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        self.logManager = LogManager()
+        super.init(coder: coder)
+    }
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.tableView.dataSource = self
+        self.tableView.register(UINib(nibName: LogListViewCell.identifier, bundle: nil), forCellReuseIdentifier: LogListViewCell.identifier)
+    }
+
+    @IBAction func didClickCloseButton(_ sender: Any) {
+        UIView.animate(withDuration: 0.5) {
+            self.view.isHidden = self.view.isHidden == true ? false : true
```

Comment:
```suggestion
            self.view.isHidden = !self.view.isHidden
```
이렇게 수정할 수 있습니다!(다른 부분도 동일)

Reply:
생각해보니, 전환이 목적인 코드라면 그렇겠군요! 감사합니다.